easybuild/easyconfigs/u/unixODBC/unixODBC-2.3.12-foss-2023a.eb

```diff
@@ -0,0 +1,27 @@
+# Easyconfig for unixODBC
+# Author: Lykle Voort
+# SURFsara, Amsterdam, The Netherlands
+# Updated: Petr Král (INUITS)
+# Updated: Stefan Wolfsheimer (SURF)
+
+easyblock = 'ConfigureMake'
+
+name = 'unixODBC'
+version = '2.3.12'
+
+homepage = "https://www.unixodbc.org"
+description = """unixODBC provides a uniform interface between
+application and database driver"""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
```

Comment:
I don't think this needs the full foss toolchain, could we use GCCcore instead (note that this will require binutils to be added as build dependency)?